virtual_rainforest/core/core_schema.json

```diff
@@ -132,36 +132,42 @@
                      "type": "boolean",
                      "default": true
                   },
-                  "out_path_initial": {
-                     "description": "File path to save initial state to",
+                  "out_path": {
+                     "description": "File path for output files",
                      "type": "string",
-                     "default": "./initial_state.nc"
+                     "default": "."
+                  },
+                  "out_initial_file_name": {
+                     "description": "File name for initial state output file",
+                     "type": "string",
+                     "default": "initial_state.nc",
+                     "pattern": "^[^/\\\\]+$"
                   },
                   "out_folder_continuous": {

```

Comment:
Did you mean to keep this option? If you did why was the default option for it removed?